Android.mk

```diff
@@ -389,8 +395,13 @@ LOCAL_STATIC_LIBRARIES += libavb_kernelflinger-$(TARGET_BUILD_VARIANT)
 ifneq ($(TARGET_BUILD_VARIANT),user)
     LOCAL_STATIC_LIBRARIES += libadb-$(TARGET_BUILD_VARIANT)
 endif
+#Replace kf4abl.c by kernelflinger.c
```

Comment:
Why do we need to replace kf4abl.c by kernelflinger.c?
Does boot logic change with this replacement?